src/main/java/frc/robot/oi/OIHandheld.java

```diff
@@ -98,38 +98,33 @@ public double getSniperLowLevel() {
 
     @Override
     public boolean getSniperHigh() {
-        return driverController.getBButton() || driverController.getBumper(Hand.kRight);
-    }
-
-    @Override
-    public boolean getSniperLow() {
-        return driverController.getAButton() || driverController.getBumper(Hand.kLeft);
+        return driverController.getBumper(Hand.kRight);
     }
 
     @Override
     public boolean hasDualSniperMode() {
         return true;
```

Comment:
We should probably not be using the dual sniper mode functionality if we only have the one button.
```suggestion
        return false;
```